### PR TITLE
Upgrade package_info_plus version

### PR DESCRIPTION
**Pull request by redDwarf03**
package_info_plus: ^6.0.0

### CODE REVIEW

**Comment by lcsvcn** [comment]
I think latest version is 7.0.0. https://pub.dev/packages/package_info_plus

That implementation works for the latest version of package_info_plus?

File: pubspec.yaml [hunk 1]
```diff
@@ -12,17 +12,14 @@ dependencies:
   flutter:
     sdk: flutter
 
-  http: 
-  package_info_plus: ^4.0.0
+  http:
+  package_info_plus: ^6.0.0
```
[the remark this answers is not in the document]
Comment (kauemurakami) [reply pubspec.yaml:16]: thanks for contribution

[the remark this answers is not in the document]
Comment (kauemurakami) [reply pubspec.yaml:16]: the latest version require update javaversion in compileOptions, i working this.

**Comment by kauemurakami** [comment]
@lcsvcn @redDwarf03 
There was a problem in `compileOptions`, please tell me the version you are using in your `example/android/app/build.gradle`.
Mine  try:
```dart
compileOptions {
  sourceCompatibility JavaVersion.VERSION_1_8
  targetCompatibility JavaVersion.VERSION_1_8
}
```

AND

```dart
compileOptions {
  sourceCompatibility JavaVersion.VERSION_17
  targetCompatibility JavaVersion.VERSION_17
}
```

The error complains :
```json 
Execution failed for task ':package_info_plus:compileDebugJavaWithJavac'.
> error: invalid source release: 17
```

I using Java 17 in my machine and accordingly [this change log ](https://pub.dev/packages/package_info_plus/changelog#600) we have update  VERSION java to 17, but my machine for the build use java 17, yet don't resolve this, that's why before launch new version, i go resolver this.
Please you could share your compile options ? 

We may have to update our gradle to adjust this to fix this warning also.
```json
  You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/go/flutter-gradle-plugin-apply

You are applying Flutter's main Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle 
plugins with the declarative plugins block: https://flutter.dev/go/flutter-gradle-plugin-apply
```
And make necessary updates.

**Comment by kauemurakami** [comment]
Indicated solution of the now https://github.com/fluttercommunity/plus_plugins/issues/2829#issuecomment-2053776864

**Comment by kauemurakami** [comment]
I fix this with update AS, I'll be checking for updates as soon as possible.

**Pull request merged**